src/Core/Command/ImportCategoryAttributesCommand.php

```diff
@@ -144,6 +155,12 @@ private function import(?string $categoryId, bool $process, SymfonyStyle $io): v
         $categoryAttributeList = $this->categoryAttributeLoader->fetch($categoryIdList);
         $io->writeln(" ... done");
 
+        if ($prune) {
+            $io->writeln('Pruning channel categories');
```

Comment:
Hier könntest du ggf. auf ein $io->write(...) gehen und das done via writeln() schreiben. Dann passt es in eine zeile